src/pages/EntityDetails/counts.js

```diff
@@ -31,5 +25,32 @@ export const generateUnitSecondaryCounts = (modelStatusData) => {
       return incrementCounts(status, counts);
     });
   });
-  return [formatCounts(counts), totalUnits];
+  return [counts, totalUnits];
+};
+
+export const renderCounts = (activeView, modelStatusData) => {
+  if (!modelStatusData) return null;
+  let chips = null;
+  switch (activeView) {
+    case "localApps":
+      chips = generateSecondaryCounts(
+        modelStatusData,
+        "applications",
+        "status"
+      );
+      break;
+    case "units":
+      [chips] = generateUnitSecondaryCounts(modelStatusData);
+      break;
+    case "machines":
+      chips = generateSecondaryCounts(
+        modelStatusData,
+        "machines",
+        "agent-status"
+      );
+      break;
+    case "relations":
+      return null;
```

Comment:
this should probably just have a `break;` here and then chips will be returned as null.